ERCS/erc-7652.md

```diff
@@ -0,0 +1,160 @@
+    /// @param tokenId Guaranteed NFT (token ID),
+    event GuaranteeSequenceIsEstablshed(
+        address userGuaranteed,
+        uint256 number,
+        address DAOs,
+        uint256 indexed tokenId
+    );
+
+    /// @notice   A user's evaluation for an NFT (token ID)
+    /// @dev      Set the guarantee information for one guarantor,
+    /// Throws if `_tokenId` is not a valid NFT
+    /// @param value  user's evaluation for  an NFT, the oledr value is canceled,
+    /// @param user   address of guarantor
+    /// @param weight guarantee weight for guarantor
+    /// @param tokenId The NFT
+    /// @return the error status of function execution
+    function setNFTGuarantedInfo(
+        uint256 value,
+        address user,
+        uint256 weight,
+        uint256 tokenId
+    ) external returns (uint256);
+
+    /// @notice   Establish guarantee sequence for an NFT (token ID) and split the commission
+    /// @dev      Each NFT(token ID) retains a current guarantee sequence,
+    ///           and expired guarantee sequences are no longer valid,
+    ///           Throws if `_tokenId` is not a valid NFT
+    /// @param valueCommission Commission for a transactions
+    /// @param userGuaranteed   address of guaranteed
+    /// @param number  block.number of transaction,
+    ///              and all DAOs established before this point will enter the guarantee sequence
+    /// @param tokenId The NFT
+    /// @return the error status of function execution
+    function establishNFTGuarantee(
+        uint256 valueCommission,
+        address userGuaranteed,
+        uint256 number,
+        uint256 tokenId
+    ) external returns (uint256);
+
+    /// @notice   Transactions that fulfill the guarantee responsibility
+    /// @dev      The new accountability transaction also requires
+    ///           the construction of a new guarantee sequence
+    ///           Throws if `_tokenId` is not a valid NFT or userGuaranteed is not right
+
+    /// @param  userGuaranteed   address of guaranteed
+    /// @param  tokenId The NFT
+    /// @return the error status of function execution
+    function FulfillGuaranteeTransfer(address userGuaranteed, uint256 tokenId)
+        external
+        returns (uint256);
+}
+
+```
+
+## Rationale
+
+Key factors influencing the standard:
+
+- Pay attention to ensuring fairness between and within groups when allocating commissions
+- Keeping the number of guarantee groups (DAOs)in the interfaces to prevent contract bloat
+- The guarantee group is a DAO contract, which MUST implement the `ERC721TokenReceiver` interface
+- Simplicity
+- Gas Efficiency
+
+
+## Backwards Compatibility
+
+This standard is compatible with current EIP-721 standards. There are no other standards that define a similar role for NFTs and the name (Guarantor) is not used by other EIP-721 related standards.
+
+
+## Reference Implementation
+
+The reference implementation will be provided later.
+
```

Comment:
```suggestion
```

Just remove this section until you have the reference implementation ready.

Reply:
Thank you for your suggestion, the new version has made the relevant revisions to the wording.

Reply:
Thank you for your suggestion, the new version has corrected this mistake. Very appreciate it.